src/main/scala/com/gradle/internal/Overrides.scala

```diff
@@ -1,15 +1,20 @@
 package com.gradle.internal
 
-import com.gradle.internal.Utils.{booleanSysPropertyOrEnvVariable, sysPropertyOrEnvVariable}
+import com.gradle.develocity.agent.sbt.api.configuration.Server
+import com.gradle.internal.Utils.Env
+import java.net.URL
 
-object Overrides {
+class Overrides(implicit env: Env) extends Transformer[Server] {
 
   // System properties to override Develocity configuration
-  private val GRADLE_ENTERPRISE_URL = "gradle.enterprise.url"
-  private val GRADLE_ENTERPRISE_ALLOW_UNTRUSTED_SERVER = "gradle.enterprise.allowUntrustedServer"
+  private val serverUrl = Env.Key[URL]("gradle.enterprise.url")
```

Comment:
was it intentional that this was changed from `develocity.url` back to `gradle.enterprise.url` ?

Reply:
Yes, the integration tests only inject `gradle.enterprise.url` into the system properties at the moment.

Reply:
@runningcode I will add (perhaps separately) handling of both `develocity.*` and `gradle.enterprise.*` for overrides, as we do for ccud gradle and maven already.

Reply:
If the only reason to add `gradle.enterprise.url` is so that integration tests will pass then that is not a good reason. We should adjust the tests.

Reply:
That's true, I'll update the tests.